plugin-server/src/worker/ingestion/event-pipeline/prepareEventStep.ts

```diff
@@ -1,12 +1,22 @@
 import { PluginEvent } from '@posthog/plugin-scaffold'
 import { PreIngestionEvent } from 'types'
 
+import { UUID } from '../../../utils/utils'
 import { parseEventTimestamp } from '../timestamps'
 import { captureIngestionWarning } from '../utils'
 import { EventPipelineRunner } from './runner'
 
 export async function prepareEventStep(runner: EventPipelineRunner, event: PluginEvent): Promise<PreIngestionEvent> {
     const { team_id, uuid } = event
+    const { db } = runner.hub
```

Comment:
I was thinking we should move this earlier to the very first step in the runner - populateTeamData